src/configstore.ts

```diff
@@ -4,6 +4,8 @@ import { AppConfigurationClient, ConfigurationSettingId, SetConfigurationSetting
 import { getErrorMessage } from './errors';
 import { Tags } from './input';
 
+const userAgentPrefix = "GitHub-AppConfiguration-Sync/1.0.0";

```

Comment:
I see that the version 1.0.0 is hardcoded. Is that available in code somehow?

Reply:
Do you mean obtaining the version dynamically at runtime? The other place that the version number will be is in the GitHub release (created when we publish the action). But I'm not aware of a good way to get the info from there. GitHub downloads the action into their virtual environment and executes it there (action doesn't have context about where it was downloaded from).